examples/pylab_examples/annotation_demo.py

```diff
@@ -91,7 +91,7 @@
 fig = plt.figure()
 ax = fig.add_subplot(111, projection='polar')
 r = np.arange(0, 1, 0.001)
-theta = 2*2*np.pi*r
+theta = 4 * np.pi * r
```

Comment:
Reverted; however I chose to leave the use of `**2` instead of `X * X` above (unless you feel strongly about it).